src/v2/Apps/Debug/DebugInquiryApp.tsx

```diff
@@ -0,0 +1,18 @@
+// TODO: Remove once inquiry is complete
+import { Button } from "@artsy/palette"
+import React from "react"
+import { useInquiry } from "v2/Components/Inquiry/useInquiry"
+
+export const DebugInquiryApp: React.FC<{}> = () => {
+  const { inquiryQuestionnaire, showInquiry } = useInquiry({
+    artworkID: "andy-warhol-the-shadow-from-myths-2",
+  })
+
+  return (
+    <>
+      <Button onClick={showInquiry}>Open inquiry questionnaire</Button>
+
+      {inquiryQuestionnaire}
+    </>
+  )
+}
```

Comment:
Rather than trying to deal with Relay etc in Storybook; just adding a simple route here that I'll teardown when it's done.